paper-menu.html

```diff
@@ -0,0 +1,330 @@
+<!--
+    @license
+    Copyright (c) 2015 The Polymer Project Authors. All rights reserved.
+    This code may only be used under the BSD style license found at http://polymer.github.io/LICENSE.txt
+    The complete set of authors may be found at http://polymer.github.io/AUTHORS.txt
+    The complete set of contributors may be found at http://polymer.github.io/CONTRIBUTORS.txt
+    Code distributed by Google as part of the polymer project is also
+    subject to an additional IP rights grant found at http://polymer.github.io/PATENTS.txt
+-->
+
+<link rel="import" href="../polymer/polymer.html">
+<link rel="import" href="../core-selector/core-selectable.html">
+<link rel="import" href="../layout/x-layout.html">
+<link rel="import" href="../paper-styles/x-theme.html">
+
+<!--
+@element paper-menu
+-->
+
+<dom-module id="paper-menu">
+
+  <style>
+
+    :host {
+      padding: 8px 0;
+
+      background: var(--primary-background-color);
+      color: var(--primary-text-color);
+
+      mixin(--paper-menu);
+    }
+
+    :host > ::content > [disabled] {
+      color: var(--disabled-text-color);
+    }
+
+    :host > ::content > .core-selected {
+      position: relative;
+    }
+
+    :host > ::content > .core-selected::after {
+      mixin(--fit);
+
+      background: currentColor;
+      /* FIXME move to paper-styles for next widget */
```

Comment:
Yeah, was thinking about fixing it once something else shares these variables which looks to be other widgets with focus states... Do you have one now?